manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -39,3 +39,27 @@ spec:
           for: 15m
           labels:
             severity: critical
+        - alert: GarbageCollectorSyncFailed
+          annotations:
+            summary: There was a problem with syncing the resources for garbage collection.
+            description: Garbage Collector had a problem with syncing and monitoring the available resources. Please see KubeControllerManager logs for more details.
+          expr: |
+            rate(garbagecollector_controller_garbagecollector_resources_sync_error_total{}[1h]) > 0
+          labels:
+            severity: warning
+        - alert: GarbageCollectorObjectDeletionFailed
+          annotations:
+            summary: There was a problem with deleting an object.
+            description: Garbage Collector had some problems when attempting to delete an object. Please see KubeControllerManager logs for more details.
```

Comment:
is it okay to write see the logs as an advice?

Reply:
Yes, it is.

The best is always to write a dedicated runbook to provide as much information as possible, but having a comprehensive description is a good start.